playwright-e2e/authentication/auth-base.ts

```diff
@@ -19,12 +19,7 @@ export async function fillInEmailPassword(
   const passwordInput = page.locator('input[type="password"]').locator('visible=true');
   const notYourAcctLink = page.locator('.auth0-lock-alternative-link:has-text("Not your account")');
 
-  const timeout = 10000;
-  await Promise.race([
-    notYourAcctLink.waitFor({ state: 'visible' }),
-    emailInput.waitFor({ state: 'visible' }),
-    new Promise((_, reject) => setTimeout(() => reject(Error('Timeout Error: Fail to confirm Log in successful.')), timeout)),
-  ]);
```

Comment:
This was what failed step look like before.

<img width="963" alt="Screenshot 2024-03-14 at 10 41 18 PM" src="https://github.com/broadinstitute/ddp-angular/assets/35533885/ad19dd0b-bf8b-4ec8-be48-475ef31f8f6b">